helper/DockerHelper.go

```diff
@@ -236,6 +236,10 @@ func BuildArtifact(ciRequest *CiRequest) (string, error) {
 		for key, value := range dockerBuildFlags {
 			dockerBuild = dockerBuild + " " + key + value
 		}
+		if dockerBuildConfig.BuildContext == "" {
+			dockerBuildConfig.BuildContext = "."
+		}
+		dockerBuildConfig.BuildContext = path.Join(".", dockerBuildConfig.BuildContext)
```

Comment:
can `dockerBuildConfig.BuildContext ` be null or empty? If yes, would this work as expected?